dice.js

```diff
@@ -0,0 +1,10 @@
+
+roll_1= Math.floor(Math.random()*6+1);
+roll_2= Math.floor(Math.random()*6+1);
```

Comment:
Why isn't this inside your function body?

Reply:
const rollDice = function() {
 roll_1= Math.floor(Math.random()*6+1);
 roll_2= Math.floor(Math.random()*6+1);

    return `Dice rolled are ${roll_1} and ${roll_2}. Sum is ${roll_1 + roll_2}`

}
rollDice(roll_1, roll_2)